src/libmongoc/src/mongoc/mongoc-topology-description.c

```diff
@@ -649,6 +654,57 @@ _mongoc_topology_description_validate_max_staleness (
    return true;
 }
 
+static bool
+_check_any_server_less_than_wire_version_13 (const void *sd_,
+                                             void *any_too_old_)
```

Comment:
I assume this signature is necessary for the `mongoc_set_for_each` API? Each callback must return true to indicate success and allow iteration to continue?

Reply:
This is correct